### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by strass**
WIP. Not very sophisticated but I'm interested in figuring out how reakit works

- [x] bootstrap styles not showing up
  - [ ] make them more bootstrappy
  - [ ] aside elements have position: absolute by default
- [x] website needs entry
  - [ ] docs need work
  - [ ] usage instructions? positioning, assertiveness, removal, etc
  - [ ] status enums? error success info warn etc
- [ ] open/visible prop on non-dialog elements
- [ ] axe doesn't like role="alert"

### CODE REVIEW

File: packages/reakit-system-bootstrap/src/Alert.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  const alert = css`
+    padding: 10px;
+    border: 2px solid hsl(206, 74%, 54%);
+    border-radius: 4px;
+    background: hsl(206, 74%, 90%);
+
+    &:empty {
+      display: none;
+    }
+  `;
```
Comment (strass) [inline packages/reakit-system-bootstrap/src/Alert.ts:20]: from https://www.w3.org/TR/wai-aria-practices-1.1/examples/alert/css/alert.css

File: packages/reakit/src/Alert/__examples__/Alert/__tests__/index-test.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import * as React from "react";
+import { render, screen, axe } from "reakit-test-utils";
+import AlertTest from "..";
+
+test("a11y", async () => {
+  const { baseElement } = render(<AlertTest />);
+  expect(await axe(baseElement)).toHaveNoViolations();
```
Comment (strass) [inline packages/reakit/src/Alert/__examples__/Alert/__tests__/index-test.tsx:7]: <pre>
FAIL packages/reakit/src/Alert/__examples__/Alert/__tests__/index-test.tsx
  ● a11y

    expect(received).toHaveNoViolations(expected)

    Expected the HTML found at $('dialog') to have no violations:

    &lt;dialog role=&quot;alert&quot; open=&quot;&quot;&gt;Alert&lt;/dialog&gt;

    Received:

    "ARIA role must be appropriate for the element (aria-allowed-role)"

    Fix any of the following:
      ARIA role alert is not allowed for given element

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/4.0/aria-allowed-role?application=axeAPI

       5 | test("a11y", async () => {
       6 |   const { baseElement } = render(<AlertTest />);
    >  7 |   expect(await axe(baseElement)).toHaveNoViolations();
         |                                  ^
       8 | });
       9 | 
      10 | test("markup", () => {

      at Object.<anonymous> (packages/reakit/src/Alert/__examples__/Alert/__tests__/index-test.tsx:7:34)
</pre>

**Comment by strass** [comment]
@diegohaz do you think this has legs? what's the baseline for a component that's worth including in the library?

**Comment by artalar** [comment]
@diegohaz can u look at it, please?

**Pull request closed without merging**